plugins/newsdownloader.koplugin/feeds.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I don't mind to have the feeds configuration in different format as long it's clear to read and we have examples there. 

I just used XML as I was already familiar with parsing that in lua. 

I was also against mixing feeds configuration with application variables.